mjidris/irondb
# Us41 t42 t43 t44 t45 t46

Condensed the optional input boxes for 'journal' and 'year' into one row, and aligned them to better match the previous row.

Moved the plus-square to the right side

*Edit*
Added commit for tasks regarding the spacing issues between the search panel and results table.
To test:
- Go to /database
- Add a random amount of optional input boxes to the search panel
- Make both successful and unsuccessful searches
- Spacing between the search panel and results table should stay consistent, regardless of the number of optional input boxes in the search panel
